README.md

```diff
@@ -138,6 +138,16 @@ WebSecurityHeaderInjector injector = new WebSecurityHeaderInjector(webSecurityBu
 ```
 
 
+### Using Outside of Dropwizard
+You can use the individual filters outside of a Dropwizard Application by depending on:
+
+```groovy
+dependencies {
+    compile 'com.palantir.websecurity:web-security:<latest-version>'
+}
+```
+
```

Comment:
It might be nice to provide a usage example. My guess is without an example nobody will use this feature